javascript/exosecurity/features/support/world.js

```diff
@@ -0,0 +1,45 @@
+const ObservableProcess = require('observable-process')
+const path = require('path')
+const { any } = require('prelude-ls')
```

Comment:
Can we remove `prelude-ls`? Feels weird to use that now that we aren't using livescript.  We can use lodash in order to have an `any` function